egs/voxforge/asr1/run.sh

```diff
@@ -210,6 +213,7 @@ if [ ${stage} -le 3 ]; then
         --maxlen-in ${maxlen_in} \
         --maxlen-out ${maxlen_out} \
         --opt ${opt} \
+        --scheduled-sampling-ratio ${samp_ratio} \
```

Comment:
The option name would be a bit too long.
Any idea?

Reply:
What about sample-probability ?

Reply:
I will keep it as sampling-probability, as it sounds good..